src/Console/ListenerMakeCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PodPoint\AwsPubSub\Console;
+
+use Illuminate\Console\GeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+
+class ListenerMakeCommand extends GeneratorCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'pubsub:make:listener';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new PubSub event listener class';
+
+    /**
+     * The type of class being generated.
+     *
+     * @var string
+     */
+    protected $type = 'Listener';
+
+    /**
+     * Get the stub file for the generator.
+     *
+     * @return string
+     */
+    protected function getStub()
+    {
+        if ($this->option('queued')) {
+            return __DIR__.'/stubs/listener-queued.stub';
+        }
+
+        return __DIR__.'/stubs/listener.stub';
+    }
+
+    /**
+     * Get the default namespace for the class.
+     *
+     * @param  string  $rootNamespace
+     * @return string
+     */
+    protected function getDefaultNamespace($rootNamespace)
```

Comment:
Can add string typing to this.

Reply:
Nope because Laravel's definition of it is not using type hinting.